mods/sa/rules/creep-plants.yaml

```diff
@@ -54,16 +54,16 @@ venus:
 		HP: 30000
 	Armament:
 		Weapon: venus
-	#WithAttackAnimation@normal:
-	#	Sequence: shoot
-	#	RequiresCondition: !injured
-	#WithAttackAnimation@injured:
-	#	Sequence: damaged-shoot
-	#	RequiresCondition: scratched
-	#GrantConditionOnDamageState: #TODO lack of support lightly damaged state
-	#	Condition: scratched
-	#	ValidDamageStates: Injured
-	#	GrantPermanently: False
+	WithAttackAnimation@normal:
+		Sequence: shoot
+		RequiresCondition: !scratched
+	WithAttackAnimation@injured:
+		Sequence: damaged-shoot
+		RequiresCondition: scratched
+	GrantConditionOnDamageState:
+		Condition: scratched
+		ValidDamageStates: Light, Medium, Heavy, Critical
```

Comment:
That means the plant will attack actors no matter what damage state it has. In SA plants are able to attack when their visual state is normal (unharmed) or lightly damaged.

Reply:
Which `DamageStates` exactly do you suggest?